test/reporters.js

```diff
@@ -1116,7 +1116,6 @@ describe('Reporter', () => {
                 expect(result.lint[0].filename).to.exist();
                 expect(result.lint[0].errors).to.exist();
                 done();
-                done = function () {};
```

Comment:
I know, but I see no way for it to be relevant unless there’s a bug in `Lab.report` that causes it to call its callback more than once.

And **if** such a case exists in `Lab.report`, I’d prefer this test to fail violently instead of shadowing the problem. Or did I miss something?